packages/seacas/libraries/ioss/src/Ioss_ChangeSet.h

```diff
@@ -0,0 +1,101 @@
+// Copyright(C) 2024 National Technology & Engineering Solutions
+// of Sandia, LLC (NTESS).  Under the terms of Contract DE-NA0003525 with
+// NTESS, the U.S. Government retains certain rights in this software.
+//
+// See packages/seacas/LICENSE for details
+
+#pragma once
+
+#include "Ioss_DatabaseIO.h"
+#include "Ioss_DBUsage.h"
+#include "Ioss_Region.h"
+#include "Ioss_ParallelUtils.h"
```

Comment:
Probably add `PropertyManager.h` include since referenced in the file...   Or could forward-declare it and `Ioss::Region`...

Reply:
I'll add the header